ERCS/erc-7656.md

```diff
@@ -1,14 +1,15 @@
 ---
 eip: 7656
 title: Generalized Contract-Linked Services
-description: Define a factory for generic services linked to any contract type including ERC-4337 accounts and NFTs
+description: Define a factory for generic services linked to any contract type, including smart accounts and NFTs
 author: Francesco Sullo (@sullof)
 discussions-to: https://ethereum-magicians.org/t/variation-to-erc6551-to-deploy-any-kind-of-contract-linked-to-an-nft/19223
-status: Review
+status: Last Call
```

Comment:
Status change should be a separate PR. Can you split this PR into two PRs? One with all the change, and the second one with the move to Last Call?

Reply:
Sure. Thanks. I will change this one for review, and make a new one with the move to last call.